root/components/home/Explore.js

```diff
@@ -0,0 +1,172 @@
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/MusicBrainz_Database"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('The Database')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  The MusicBrainz Database stores all of the
+                  various pieces of information we collect
+                  about music, from artists and their
+                  releases to works and their composers,
+                  and much more.
+                  All our data is available for commercial licensing.
+                  If you are interested in licensing this data
+                  for commercial use, please contact us.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/Editing_FAQ"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Breadth')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  MusicBrainz exists to collect as much
+                  information about music as we can. We do not
+                  discriminate or prefer one “type” of
+                  music over another. Whether it is published or
+                  unpublished, popular or fringe, western or non-western,
+                  human or non-human — we want it all in MusicBrainz.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/How_Editing_Works"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Editing Philosophy')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  Maintaining a comprehensive database of all types
+                  of music is a large task, and MusicBrainz depends
+                  on its users to spot mistakes and correct them.
+                  Our editing and voting system gives users the ability to
+                  update our data and review others’ changes.
```

Comment:
Maybe "MusicBrainz depends on its users not just to add the data, but also to spot mistakes and correct them"? Otherwise it sounds a bit like the data comes from elsewhere.

Also "Our editing and voting systems give users the ability to both update the  data and review each others’ changes." maybe?